app/routes/edit.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  model() {
+    return this.get('store').findRecord('reminder', this.model.id);
+  },
+
+  action: {
+    saveReminder() {
+      this.transitionTo("/reminders")
```

Comment:
It isn't, and is completely unnecessary at this point. We had originally tried handling it in the route, but instead used a link in the edit-reminder component. Good eyes!